docs/D-Modularity/library-functions.md

```diff
@@ -139,14 +143,14 @@ The following program outputs a different set of 10 pseudo-random numbers with e
 ```c
 #include <stdlib.h>
 #include <stdio.h>
-#include <time.h> // prototype for time(NULL)
+#include <time.h> // ...prototype for time(NULL)...
 
 int main(void)
 {
         int i;
 
-        srand(time(NULL)); // will set a unique seed for each run of the program
-        for (i = 0; i < 10 ; i++)
+        srand(time(NULL)); // ...will set a unique seed for each run of the program...
```

Comment:
Remove ...

Reply:
I removed extra ... from comments in the new commit. Thank you for reviews 👍